x-pack/plugins/security_solution/public/resolver/view/clickthrough.test.tsx

```diff
@@ -136,6 +137,11 @@ describe('Resolver, when analyzing a tree that has two related events for the or
   });
 
   describe('when it has loaded', () => {
+    let originBounds: ReturnType<typeof getComputedNodeBoundaries>;
```

Comment:
Would you change this to `AABB` for readability?
```suggestion
    let originBounds: AABB;
```